website/promotion/services.py

```diff
@@ -0,0 +1,13 @@
+from django.utils import timezone
+from promotion.models import PromotionRequest
+
+def get_new_requests(end_date):
```

Comment:
It would be nicer to implement this as a custom ModelManager, like in here for example https://docs.djangoproject.com/en/4.0/topics/db/managers/